crates/ln-dlc-node/src/node.rs

```diff
@@ -814,23 +828,45 @@ pub(crate) fn app_config() -> UserConfig {
 pub(crate) fn coordinator_config() -> UserConfig {
     UserConfig {
         channel_handshake_config: ChannelHandshakeConfig {
+            // The coordinator will by default only accept public channels. (see also
+            // force_announced_channel_preference). In order to open a private channel with the
+            // mobile app this config gets overwritten during the creation of the just-in-time
+            // channel)
+            // Note, public channels need 6 confirmations to get announced (and usable for multi-hop
+            // payments) this is a requirement of BOLT 7.
             announced_channel: true,
+            // The minimum amount of confirmations before the inbound channel is deemed useable,
+            // between the counterparties
             minimum_depth: 1,
             // only 10% of the total channel value can be sent. e.g. with a volume of 30.000 sats
             // only 3.000 sats can be sent.
             max_inbound_htlc_value_in_flight_percent_of_channel: 10,
             ..Default::default()
         },
         channel_handshake_limits: ChannelHandshakeLimits {
+            // The minimum amount of confirmations before the outbound channel is deemed useable,
+            // between the counterparties
             max_minimum_depth: 1,
             trust_own_funding_0conf: true,
-            force_announced_channel_preference: false,
+            // Enforces incoming channels to the coordinator to be public! We
```

Comment:
Because of this, do we need to ensure that a routing hint is always added to an invoice generated by the app?

Reply:
Not sure if I fully understand the context of your question. But yes I think we will always need to add a routing hint to the invoice if we want to send a payment over a private channel.